sdk/dnsresolver/Azure.ResourceManager.DnsResolver/api/Azure.ResourceManager.DnsResolver.net8.0.cs

```diff
@@ -571,7 +574,7 @@ protected DnsSecurityRuleCollection() { }
     public partial class DnsSecurityRuleData : Azure.ResourceManager.Models.TrackedResourceData, System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.DnsResolver.DnsSecurityRuleData>, System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.DnsResolver.DnsSecurityRuleData>
     {
         public DnsSecurityRuleData(Azure.Core.AzureLocation location, int priority, Azure.ResourceManager.DnsResolver.Models.DnsSecurityRuleAction action, System.Collections.Generic.IEnumerable<Azure.ResourceManager.Resources.Models.WritableSubResource> dnsResolverDomainLists) { }
-        public Azure.ResourceManager.DnsResolver.Models.DnsSecurityRuleAction Action { get { throw null; } set { } }
+        public Azure.ResourceManager.DnsResolver.Models.DnsSecurityRuleActionType? ActionType { get { throw null; } set { } }
```

Comment:
as far as I could see here, this property changes its name, and become an optional property.
This is a breaking change that is simple to mitigate therefore we do not need to bump the major version for such a minor change.

Reply:
so by offline sync, I see this change was caused by that we removed a property on the `DnsSecurityRuleAction` model.
And now this model only has one property which triggers the mechanism in the generator to "flatten" it. In the meantime, the generator will hide the original property.
Therefore we saw this change, which is actually: the previous `DnsSecurityRuleAction Action` property is changed to internal, and a new public property `DnsSecurityRuleActionType? ActionType` was added.